### PR TITLE
Update chunk_size.md - write conclusion

### PR DESCRIPTION
**Pull request by carolina-museum**
I added the conclusion part to this document. Also updated the chunk size options that we evaluated.

### CODE REVIEW

File: documents/chunk_size.md [hunk 1]
```diff
@@ -46,4 +45,18 @@ Here are the questions:
...
+A sentence was too short for LLM to understand the context. It gave several unrelated wrong answers.
+Half a paragraph was too long, so a similar chunk to the question was incorrectly picked.
+A paragraph was too long, it distracted LLM from the main point of the question and answer.
+
+Quarter a paragraph also gave answers mixed with incorrect information, but this one had fewer incorrect answers and more correct answers.
+A quarter of a paragraph includes 1-4 sentences each.
+
+Here is an insight from reading the responses:
+The degree of personalization in response depends greatly on the prompt you give to the LLM model. Some gave general answers that were not incorrect but not aligned with the author's writing.
+We may need to specify in the prompt to what degree we want the response to be personalized.
```
Comment (huyfififi) [inline documents/chunk_size.md:62]: Yes, I see that many AI engineers out there work on prompt engineering to improve output accuracy

**Review by huyfififi — APPROVED** [review]
Awesome 👍 
Please create a GitHub issue once you decide to work on the chunking algorithm for our system.

**Review by gomyway1216 — APPROVED** [review]

File: documents/chunk_size.md (hunk 1 above)
Comment (carolina-museum) [reply documents/chunk_size.md:62]: I think I will also need to look into prompt engineering at some point then! Thank you for the information

**Comment by carolina-museum** [comment]
I got it! Thank you!

**Pull request merged**